src/comments.ts

```diff
@@ -90,7 +90,7 @@ export const PingReviewersTooMany = (names: readonly string[]) => ({
 export const PingStaleReviewer = (reviewedAbbrOid: string, reviewers: string[]) => ({
     tag: `stale-ping-${sha256(reviewers.join("-")).substr(0, 6)}-${reviewedAbbrOid}`,
     status: txt`
-        |@${reviewers.join(", @")} Thank you for reviewing this PR! The author has pushed new
+        |**${reviewers.join(", ")}** Thank you for reviewing this PR! The author has pushed new
```

Comment:
This is a way to mention people without actuating pinging them. It could be used anywhere.